Doc/library/argparse.rst

```diff
@@ -182,6 +182,10 @@ ArgumentParser objects
    .. versionchanged:: 3.5
       *allow_abbrev* parameter was added.
 
+   .. versionchanged:: 3.8
```

Comment:
```suggestion
   .. versionchanged:: 3.9
```

Reply:
I'm still hoping that this will be merged into the 3.8 branch before 3.8.0, but I guess we can adjust the number later.